src/config/ConfigManager.cpp

```diff
@@ -478,6 +478,7 @@ CConfigManager::CConfigManager() {
     m_pConfig->addConfigValue("master:mfact", {0.55f});
     m_pConfig->addConfigValue("master:new_status", {"slave"});
     m_pConfig->addConfigValue("master:slave_count_for_center_master", Hyprlang::INT{2});
+    m_pConfig->addConfigValue("master:center_master_slaves_on_right", Hyprlang::INT{0});
```

Comment:
this should default to 1 no?

Reply:
I wanted , to keep default behavior before the option , so people wouldn't really have to change their config if they want to keep the old behavior.

Reply:
then why is it 1 in config descriptions

Reply:
Becouse I am obviously blind.